ui/pages/confirmations/components/transaction-alerts/transaction-alerts.js

```diff
@@ -81,6 +82,7 @@ const TransactionAlerts = ({
   return (
     <div className="transaction-alerts">
       <BlockaidBannerAlert txData={txData} />
+      <STXBannerAlert />
```

Comment:
Once possible, can you please add a screenshot with this in the PR description?

Reply:
The comment below this one shows the UX from before and after migration with settings and transaction screen & alert before and after dismissal.